help/en/html/doc/Technical/ReleaseProcess.md

```diff
@@ -52,21 +59,21 @@ How do we decide when to do that if we don't have a monthly cadence?
  
  - How often (on what basis) should we be creating "Major Change" branch releases, hence updating that digit? Since we distribute applications to users, who are primarily _application_ users, most major changes may not even be visible.  We need to get them out to people before too much change accumulates, but too many might discourage people.  Maybe every three months? Six months?
 
- - What process is needed to identify when a release is good enough as one that should be made the default? Both the large scale, so we can talk in advance about the "2020-Oct" release, and at the event, as we try to get something usable to converge. For the last few years, we've needed several test releases to converge on a quality level; June 2020 was no different in that respect. How do we get that done in this model?
+ - What process is needed to identify when a release is good enough as one that should be made the default? Both the large scale, so we can talk in advance about the "October 2020" (2020-10) release, and at the event, as we try to get something usable to converge. For the last few years, we've needed several test releases to converge on a quality level; June 2020 was no different in that respect. How do we get that done in this model?
  
 ## Other questions?
 
  - What should populate the web? 
    - By doing that from `master`, it'll automatically be stable; some people like that.
-   - We don't know a-priori what will be released next, so we can't populate it from the _next__ changes as they're developed
+   - We don't know a-priori what will be released next, so we can't populate it from the _next_ changes as they're developed
    - If desired, we could have all four (master, dev-major, dev-minor, dev-branch) on the web at slightly different URLs; that just moves the question to which should be the default
     
 
  ## Other Things to Note
  
 JMRI has, for a long time, followed a Linux-like release numbering system where odd-numbered minor releases were for development and tests, whilst even numbered releases were for production. That distinction is no longer present here.
 
-There is no longer an explicit [deprecation cycle](https://www.jmri.org/help/en/html/doc/Technical/RP.shtml#deprecating). One can certain mark parts of the API as deprecated in a update or minor change; that's polite.  But when the change gets into the major branch, the deprecations should have bee removed:  A breaking change is a breaking change.
+There is no longer an explicit [deprecation cycle](https://www.jmri.org/help/en/html/doc/Technical/RP.shtml#deprecating). One can certain mark parts of the API as deprecated in a update or minor change; that's polite.  But when the change gets into the major branch, the deprecations should have been removed:  A breaking change is a breaking change.
```

Comment:
I don't know that "A breaking change is a breaking change" means that *all* deprecated code should be removed; we can retain a policy that deprecated code will be retained for a minimum of N major releases (e.g. if N == 2, code deprecated in 6.0.3 would not be removed until after 8.0 has been branched from master, when it can be removed from master in a breaking change PR, but retained on the 8.0 branch).

Reply:
I don't understand this. Perhaps I've missed something. If the deprecated code is still present, why wouldn't this be I.++J.K instead of a ++I.J.K major release?

Reply:
I think I’m trying to say that a major change should not imply that all previously deprecated code can be dropped, but only that code deprecated more than N major releases previously can be dropped.

Reply:
OK, I see your point.  I think that's really about the criteria for _merging_ a breaking PR. Does it have to be related to previous changes that set the groundwork, etc? 

I'll move this back to the jmri-developers list for discussion.